src/classes/VOL_CTRL_VolunteersJobListingFS.cls

```diff
@@ -270,7 +271,7 @@ global virtual with sharing class VOL_CTRL_VolunteersJobListingFS {
                         Location_Street__c, Location_City__c, Location__c, Location_Zip_Postal_Code__c, External_Signup_Url__c,
                         (Select Id, Start_Date_Time__c, Duration__c, Number_of_Volunteers_Still_Needed__c, Total_Volunteers__c,
                             Description__c, System_Note__c From Volunteer_Job_Slots__r 
-                            where Start_Date_Time__c >= :dtNow and Start_Date_Time__c < :dtLast
+                            //where Start_Date_Time__c >= :dtNow and Start_Date_Time__c < :dtLast
```

Comment:
I'm concerned that by removing this date filter, you could be returning too many records in this soql query for a job that has daily shifts, and has been in place for several years.  Also, it appears that not only are you showing all past shifts, you are also getting all future shifts, rather than respecting the dtLast value that is specified thru URL parameters.